cogs/music.py

```diff
@@ -365,29 +368,38 @@ async def playlist(self, ctx: commands.Context, action: str = 'show', song_numbe
             await ctx.send(boxed_string(string))
 
         elif action == 'add':
-            if song_number == 'random':
-                song_number = random.randint(0, len(_songlist) - 1)
-            _playlist.append(_songlist[int(song_number) - 1])
-            await ctx.send(boxed_string(
-                _('«‎{}»‎ added to queue.').format(
-                    _songlist[int(song_number) - 1][: -len(MUSIC_EXT)]
-                )
-            ))
+            if song_identifier == 'random':
+                song_identifier = random.randint(0, len(_songlist) - 1)
+            elif song_identifier.startswith('http'):
+                _playlist.append(song_identifier)
+                song_info = YoutubeDL({'format': 'bestaudio'}).extract_info(song_identifier, download=False)
+                await ctx.send(boxed_string(
+                    _('«‎{}»‎ added to queue.').format(
+                        song_info['title']
+                    )
+                ))
```

Comment:
![image](https://user-images.githubusercontent.com/28691436/122394788-0664d600-cf7f-11eb-8220-3123962deed3.png)

Reply:
Well, this should be fixed, no?
Make it catch an exception and warn the user about an incorrect URL.
Maybe also make it so that the link has to start with `youtube.com` or `youtu.be` to avoid exceptions altogether.